config/i18n.json

```diff
@@ -628,6 +628,7 @@
     "Owned": "This item is in your inventory.",
     "OwnedMod": "This mod is in your modifications inventory.",
     "PullItem": "Pull from {{bucket}} to {{store}}",
+    "PullPostmaster": "Pull from Postmaster",
```

Comment:
Would it be too concise to just label this "Pull"?

Reply:
i feel like the line above demonstrates adequately why i went for clarity